rollup.config.js

```diff
@@ -29,6 +30,48 @@ export default [
                 name: 'posthog',
             },
         ],
+        plugins: [
+            ...plugins,
+            copy({
+                targets: [
+                    {
+                        src: [
+                            'node_modules/rrweb/dist/record/rrweb-record.min.js',
+                            'node_modules/rrweb/dist/plugins/console-record.min.js',
+                        ],
+                        file: 'dist/recorder.js',
+                    },
+                    {
+                        src: 'node_modules/rrweb/dist/record/rrweb-record.min.js.map',
+                        dest: 'dist/',
+                        rename: 'recorder.js.map',
+                    },
+                    {
+                        src: 'node_modules/rrweb/dist/plugins/console-record.min.js.map',
+                        dest: 'dist/',
+                        rename: 'console-record.min.js.map',
+                    },
+                    {
+                        src: 'dist/recorder.js',
+                        dest: 'dist/',
+                        transform: (contents) =>
+                            contents.toString().replace('rrweb-record.min.js.map', 'recorder.js.map'),
+                    },
```

Comment:
I wonder if this is the right approach vs creating our own `loader-recorder.ts` and importing / exporting what we want by default? That way we don't have to any complex copying of the minified JS but rather just build our own bundle.

That way we could have the recorder.js exported as a module as well for those who want to import it via npm like

```js
import "posthog-js/recorder";
import PostHog from "posthog-js";
```

Not 100% sure but feels like it would simplify the rollup config as well.